easybuild/easyconfigs/r/Rosetta/Rosetta-3.5-ictce-4.1.13.eb

```diff
@@ -13,4 +13,8 @@ sources = ['%(namelower)s%(version)s_bundles.tgz']
 
 builddependencies = [('SCons', '2.3.0', '-Python-2.7.3')]
 
+modextravars = {
+    'ROSETTA3_DB': '$root/rosetta_database',
```

Comment:
can we add the validation one-liners in an area such as `./easybuild/validation/script1.sh` or such?